src/App.tsx

```diff
@@ -1,10 +1,35 @@
-import React from 'react';
+import React from "react";
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Redirect,
+} from "react-router-dom";
+import Editor from "./pages/Editor/Editor";
+import ErrorPage from "./pages/ErrorPage/ErrorPage";
+import Loader from "./pages/Loader/Loader";
 
 function App() {
   return (
-    <div >
-      Hello world!
-    </div>
+    <>
+      <header>Photo Editor</header>
+      <Router>
+        <Switch>
+          <Route exact path="/">
```

Comment:
You are pointing just for one address ("domain.com/" - nothing after .com or slash) so you do not need to use redirect. You could write <Route exact path="/" component={Loader} /> or as you wrote below: <Route exact path="/"><Loader /></Route>